CMakeLists.txt

```diff
@@ -289,18 +333,20 @@ if(BUILD_TESTING)
   target_compile_definitions(UnitTest PUBLIC
       GTEST_HAS_TR1_TUPLE=0
       GTEST_USE_OWN_TR1_TUPLE=0)
+
   target_link_libraries(
-      UnitTest PRIVATE testutils GTest::main)
+      UnitTest PRIVATE testutils GTest::main ${JAEGERTRACING_LIB})
   add_test(NAME UnitTest COMMAND UnitTest)
 
   if(TARGET jaegertracing)
     add_executable(DynamicallyLoadTracerTest
       src/jaegertracing/DynamicallyLoadTracerTest.cpp)
+
     target_include_directories(DynamicallyLoadTracerTest PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
       $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/src>)
     target_link_libraries(
-      DynamicallyLoadTracerTest OpenTracing::opentracing-static GTest::main)
+      DynamicallyLoadTracerTest OpenTracing::opentracing GTest::main)
```

Comment:
Please revert for non-Windows. I remember specifically having issues with this on Mac and/or Linux.

Reply:
change undone for Linux.